### DISCUSSION TITLE
How to calculate water balance in flopy, can you give me a example?

### DISCUSSION THREAD

**Discussion by Alvin-prc** [modflowpy/flopy#2203] [Q&A] [answered]
Water balance and solute balance are crucial indicators for evaluating the correctness of a groundwater model. In FloPy, how can i verify these balances?

**Comment by Alvin-prc** [comment]
Any body can help me with this?

**Reply by martclanor** [reply, marked as answer]
Perhaps this example can help you: https://flopy.readthedocs.io/en/3.6.0/Notebooks/zonebudget_example.html

**Reply by Alvin-prc** [reply]
Thank you
